### DISCUSSION TITLE
Upgrade to 3.10.0.CR1 Throw Exception

### DISCUSSION THREAD

**Discussion by hantsy** [quarkusio/quarkus#40186] [Q&A] [answered]
My example project:https://github.com/hantsy/quarkus-sandbox/tree/master/restclient-classic

The dependencies in the POM:

```xml
<dependency>
    <groupId>io.quarkus</groupId>
    <artifactId>quarkus-arc</artifactId>
</dependency>
<dependency>
    <groupId>io.quarkus</groupId>
    <artifactId>quarkus-resteasy</artifactId>
</dependency>
<dependency>
    <groupId>io.quarkus</groupId>
    <artifactId>quarkus-resteasy-jsonb</artifactId>
</dependency>

<dependency>
    <groupId>io.quarkus</groupId>
    <artifactId>quarkus-rest-client</artifactId>
</dependency>
<dependency>
    <groupId>io.quarkus</groupId>
    <artifactId>quarkus-rest-client-jsonb</artifactId>
</dependency>
```

```bash 
java.lang.RuntimeException: 
java.lang.RuntimeException: 
io.quarkus.builder.ChainBuildException: 
Multiple producers of item class 
io.quarkus.security.spi.DefaultSecurityCheckBuildItem 
(io.quarkus.resteasy.reactive.common.deployment.ResteasyReactiveCommonProcessor#setUpDenyAllJaxRs)

```

**Comment by hantsy** [comment]
And the https://github.com/hantsy/quarkus-sandbox/tree/master/restclient-jaxrs, https://github.com/hantsy/quarkus-sandbox/tree/master/restclient-kotlin got the same exceptions.

**Comment by gsmet** [comment, marked as answer]
For RESTEasy Classic, you need to use:
- `quarkus-resteasy`
- `quarkus-resteasy-client`

For Quarkus REST (formerly RESTEasy Reactive):
- `quarkus-rest`
- `quarkus-rest-client`

See:
- https://github.com/quarkusio/quarkus/wiki/Migration-Guide-3.7#rest-client
- https://github.com/quarkusio/quarkus/wiki/Migration-Guide-3.9#resteasy-reactive-extensions-renamed-to-quarkus-rest-gear-white_check_mark
